### PR TITLE
Pluto Input Error Logging

### PR DESCRIPTION
**Pull request by DeaBardhoshi**
## Changes:
- Added a file checker to pluto_input_pts and pluto_input_cama to output whether a file exists or not for each file

Closes Issue #47 

Relevant Runs:
- https://github.com/NYCPlanning/data-engineering/actions/runs/5797906374/job/15714498759
- https://github.com/NYCPlanning/data-engineering/actions/runs/5797908542/job/15714506358

### CODE REVIEW

**Comment by fvankrieken** [comment]
Can you add links to workflow runs showing desired behavior?

**Comment by fvankrieken** [comment]
So the pts file actually exists currently, so that one should be passing. I'll add some notes to the issue

**Review by fvankrieken — DISMISSED** [review]
I've left comments both here and on #47, happy to chat if more direction is needed

**Comment by fvankrieken** [comment]
[Cama run cancelling due to missing file](https://github.com/NYCPlanning/data-engineering/actions/runs/5871053497)

[PTS run succeeding](https://github.com/NYCPlanning/data-engineering/actions/runs/5871054516)

File: .github/workflows/pluto_input_cama.yml [hunk 1]
```diff
@@ -41,12 +41,33 @@ jobs:
...
+
+      - name: Setup ssh
+        run: ./pluto ssh setup
+
+      - name: Download file
+        id: download
+        run: |
+          (./pluto ssh download cama) && true
+          output=$?
+          if [ $output -ne 0 ] && [ $output -ne 2 ]; then
```
Comment (fvankrieken) [inline .github/workflows/pluto_input_cama.yml:53]: exit code 2 -> file not found error.

Anything else, should actually exit

File: .github/workflows/pluto_input_cama.yml [hunk 2]
```diff
@@ -41,12 +41,33 @@ jobs:
...
-        
-      - name: install ssh key
+
+      - name: Setup ssh
+        run: ./pluto ssh setup
+
+      - name: Download file
+        id: download
+        run: |
+          (./pluto ssh download cama) && true
```
Comment (fvankrieken) [inline .github/workflows/pluto_input_cama.yml:51]: The `&& true` is a bit hacky, but it makes bash not exit - within pluto cli we use `set -e` so just running this command causes the workflow run to fail

**Review by damonmcc — APPROVED** [review]
really nice having the grey cancel icon rather than the red fail

just a question for later: would it be worth combining the workflows into one? if they each take a while to download though, it might not be worth it

**Comment by fvankrieken** [comment]
Then the cancelling logic doesn't make senes - PTS is always there, CAMA only periodically

**Pull request merged**